internal/restapi/browse/helpers.go

```diff
@@ -33,6 +33,7 @@ func createModuleMetadataResponse(moduleMetadata *services.ModuleMetadata, modul
 		Provider:     moduleMetadata.Provider,
 		Description:  moduleMetadata.Description,
 		SourceUrl:    moduleMetadata.SourceUrl,
+		Maturity:     moduleMetadata.Maturity.String(),
```

Comment:
I've approved this but I think we should add a unit test to make sure this is set as expected.